style.css

```diff
@@ -16,14 +16,19 @@ img {
 .nav {
     position: relative;
     display: flex;
-    width: 58px;
-    height: 11px;
-    left: 811px;
-    top: 29px;
-    color: blanchedalmond;
+    align-items: center;
+    text-align: right;
+    color: #ffffff;
     flex-direction: row;
+    flex-flow: row;
+    width: 100%;
+    justify-content: flex-end;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
These fonts won't be applied.

Please read/watch these articles/videos about different ways how to add a font to your web page:
1. https://htmlacademy.ru/blog/boost/tutorial/fonts-loading
2. https://idg.net.ua/blog/uchebnik-css/ispolzovanie-css/shrifty-podklyuchenie-web-shriftov
3. https://www.youtube.com/watch?v=H87YikfVyHQ
4. https://www.youtube.com/watch?v=9XIPdG2B32Q